frontends/web/src/routes/account/account.tsx

```diff
@@ -61,11 +65,23 @@ export function Account({
   const [transactions, setTransactions] = useState<accountApi.TTransactions>();
   const [usesProxy, setUsesProxy] = useState<boolean>();
   const [insured, setInsured] = useState<boolean>(false);
+  const [uncoveredFunds, setUncoveredFunds] = useState<string[]>([]);
   const [stateCode, setStateCode] = useState<string>();
   const supportedExchanges = useLoad<SupportedExchanges>(getExchangeBuySupported(code), [code]);
 
   const account = accounts && accounts.find(acct => acct.code === code);
 
+  const checkUncoveredUTXOs = useCallback(async () => {
+    let uncoveredScripts: accountApi.ScriptType[] = [];
+    const utxos = await accountApi.getUTXOs(code);
+    utxos.forEach((utxo) => {
+      if (utxo.scriptType !== 'p2wpkh' && !uncoveredScripts.includes(utxo.scriptType)) {
+        uncoveredScripts.push(utxo.scriptType);
+      }
+    });
+    setUncoveredFunds(uncoveredScripts.map(script => getScriptName(script)));
```

Comment:
`.map(getScriptName)` should work as well

Reply:
cool! thanks